src/lib.rs

```diff
@@ -10,12 +10,12 @@ pub enum Interface {
     Contactless,
 }
 
-pub type Data<const S: usize> = heapless::Vec<u8, S>;
```

Comment:
`pub use data::Data` (where `data::Data` is the current `somebytes::Bytes`)? There's already a bunch of `Bytes` types in the eco-system, if we're rolling our own it would seem to make sense to name it semantically.